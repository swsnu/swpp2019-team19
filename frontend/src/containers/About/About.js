import React, { Component } from 'react';

export default class About extends Component {
  render() {
    return (
      <div>
        <p className='test'>About page</p>
      </div>
    );
  }
}
