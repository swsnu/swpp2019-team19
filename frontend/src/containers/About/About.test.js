import React from 'react';
import About from './About';
import { shallow } from 'enzyme';

describe('<About />', () => {
  it('renders', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
