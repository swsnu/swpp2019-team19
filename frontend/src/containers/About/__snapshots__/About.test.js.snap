// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders 1`] = `
<div>
  <p>
    About page
  </p>
</div>
`;
